src/blueprints/tv2/action-factories/tv2-split-screen-action-factory.ts

```diff
@@ -247,7 +247,7 @@ export class Tv2SplitScreenActionFactory {
 
   private createInsertToInputActionsForSources(blueprintConfiguration: Tv2BlueprintConfiguration, sources: Tv2SourceMappingWithSound[], name: string, audioMode: Tv2AudioMode = Tv2AudioMode.FULL): Tv2SplitScreenInsertSourceInputAction[] {
     const actions: Tv2SplitScreenInsertSourceInputAction[] = []
-    for (let inputIndex = 0; inputIndex < NUMBER_OF_SPLIT_SCREEN_BOXES; inputIndex++) {
+    for (let inputIndex = 1; inputIndex < NUMBER_OF_SPLIT_SCREEN_BOXES; inputIndex++) {
```

Comment:
This doesn't actually fix the issue.
The issue is that what the producers thinks is box number one, is actually box number zero, because the input boxes are _also_ zero-indexed. 
By starting the loop on one, you don't change that. You just don't have an Action that can insert into the first box (box zero) anymore. The Action that says it inserts into box one still inserts into box two.

The solution is to keep the index as is, but just "bump" the number when we display it.
Or do `inputIndex: inputIndex -1` when you create the `metadata` for this Action.